### DISCUSSION TITLE
how to put dnstap DNS query and response log  into one log event

### DISCUSSION THREAD

**Discussion by slgray** [vectordotdev/vector#15186] [Q&A] [answered]
Currently a pair of DNS query and response is in two event, I want to merge the pair of log into one event. I'm new to vector, any help is appreciate.

**Comment by jszwedko** [comment, marked as answer]
Hi @slgray ! I'm actually not super familiar with that source, but if it is currently emitting two events, it'd be difficult to join them later in Vector. Would you want to open a feature request to emit them as one event?

**Reply by slgray** [reply]
Thanks @jszwedko , I will open a feature request.
